test/fixtures/hello-cloud/unmanaged-pod-1.yml.erb

```diff
@@ -3,7 +3,7 @@ kind: Pod
 metadata:
   name: unmanaged-pod-1-<%= deployment_id %>
   annotations:
-    kubernetes-deploy.shopify.io/timeout-override: "60s"
+    kubernetes-deploy.shopify.io/timeout-override: 60s
```

Comment:
Maybe I should leave this file alone and just fix the expected output in the tests?

Reply:
Doesn't really matter to me